src/wdio/conf.js

```diff
@@ -10,6 +10,12 @@ exports.config = {
   capabilities: [
     {
       browserName: 'chrome',
+      axe: {
+        inject: true,
+        rules: {
```

Comment:
this should probably be 
```
options: {
  rules: {}
}
```

This allows the global options to be separated from our custom options (e.g. inject)

Reply:
updated to place the rules inside an options hash